agent/stats/engine_test.go

```diff
@@ -190,11 +190,21 @@ func TestStatsEngineMetadataInStatsSets(t *testing.T) {
 	engine.addAndStartStatsContainer("c1")
 	ts1 := parseNanoTime("2015-02-12T21:22:05.131117533Z")
 	ts2 := parseNanoTime("2015-02-12T21:22:05.232291187Z")
+	netStats := &NetworkStats{
```

Comment:
why not use createFakeContainerStats() for this test data?

Reply:
good point, not sure why this was not used before. will change it